guides/release/models/customizing-adapters.md

```diff
@@ -204,22 +208,23 @@ Requests for `user-profile` would now target `/user_profile/1`.
 Some APIs require HTTP headers, e.g. to provide an API key. Arbitrary
 headers can be set as key/value pairs on the `JSONAPIAdapter`'s `headers`
 object and Ember Data will send them along with each ajax request.
-(Note that we set headers in `init()` because default property values
+(Note that we set headers in `constructor()` because default property values
```

Comment:
Wouldn't using native classes supersede this implementation quirk ? IIRC, this pattern is because of `EmberObject.extend`. Asked in discord dev-ember-learing